layouts/embed-page/single.html

```diff
@@ -19,7 +19,7 @@
 {{ end }}
 
 {{ define "main" }}
-  <div class="bg-white p-px" data-ga-category="embed" data-fru-campaign="embed">
+  <div class="bg-g-2 p-px" data-ga-category="embed" data-fru-campaign="embed">
```

Comment:
This should be white still, because it needs to blend in on other people's article wells. The inside of the embed can be gray 2 though.